ansible/roles/etcd/handlers/main.yml

```diff
@@ -4,11 +4,9 @@
 
 - name: restart etcd
   service: name=etcd state=restarted
-  when: etcd_started.changed == false
 
 - name: restart etcd2
```

Comment:
No longer needed if the ``restart etcd`` is triggered at the end of ``tasks/main.yml``